runtimes/parachain/src/lib.rs

```diff
@@ -419,22 +408,8 @@ parameter_types! {
 	pub const MaxCandidateIntake: u32 = 1;
 }
 
-impl pallet_society::Config for Runtime {
-	type PalletId = SocietyPalletId;
-	type Event = Event;
-	type Currency = Balances;
-	type Randomness = RandomnessCollectiveFlip;
-	type CandidateDeposit = CandidateDeposit;
-	type WrongSideDeduction = WrongSideDeduction;
-	type MaxStrikes = MaxStrikes;
-	type PeriodSpend = PeriodSpend;
-	type MembershipChanged = ();
-	type RotationPeriod = RotationPeriod;
-	type MaxLockDuration = MaxLockDuration;
-	type FounderSetOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, CouncilCollective>;
-	type SuspensionJudgementOrigin = pallet_society::EnsureFounder<Runtime>;
-	type ChallengePeriod = ChallengePeriod;
-	type MaxCandidateIntake = MaxCandidateIntake;
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+	items as Balance * 20 * DOLLARS + (bytes as Balance) * 100 * MILLICENTS
```

Comment:
Where are these values coming from?

Reply:
It's copy pasted from the governance config from kusama. It doesn't really matter since it's for the election phragmen pallet which we only have in here as a place holder at the moment.

Reply:
We didn't really put much thought into any of these values yet. I think the values parity uses are sensible defaults and we will check and reevaluate these values when we tackle this: https://github.com/KILTprotocol/ticket/issues/1194